### PR TITLE
feat: add package1Display method for use with package1:version:displa…

### PR DESCRIPTION
**Pull request by WillieRuemmele**
…y command

@W-10833140@

adds `package1Display` method for querying org for package1 commands - https://github.com/salesforcecli/plugin-packaging/pull/26

### CODE REVIEW

File: src/package1/packageDisplay.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+/**
+ * Executes server-side logic for the package1:display command
+ *
+ * @param connection
+ * @param id: MetadataPackageVersion
+ */
+export async function package1Display(connection: Connection, id: string): Promise<Package1Display[]> {
+  const query = `SELECT Id,MetadataPackageId,Name,ReleaseState,MajorVersion,MinorVersion,PatchVersion,BuildNumber FROM MetadataPackageVersion WHERE id = '${id}'`;
```
Comment (peternhale) [inline src/package1/packageDisplay.ts:26]: @WillieRuemmele Could you create `MetadataPackageVersion` type and expose it in /src/interfaces/packagingSobject.ts.

File: src/package1/packageDisplay.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const results = (await connection.tooling.query(query)).records;
```
Comment (peternhale) [inline src/package1/packageDisplay.ts:27]: Then the type can be added to the query 
``` suggestion
  const results = (await connection.tooling.query<PackagingSobjects.MetadataPackageVersion>(query)).records;
```

File: src/package1/packageDisplay.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Connection } from '@salesforce/core';
+
+export type Package1Display = {
```
Comment (peternhale) [inline src/package1/packageDisplay.ts:10]: Would you move this to src/interfaces/packagingInterfacesAndType.ts

**Review by peternhale — CHANGES_REQUESTED** [review]

**Review by peternhale — APPROVED** [review]

File: src/package1/packageDisplay.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    return {
+      MetadataPackageVersionId: result.Id,
+      MetadataPackageId: result.MetadataPackageId,
+      Name: result.Name,
+      ReleaseState: result.ReleaseState,
+      Version: `${result.MajorVersion}.${result.MinorVersion}.${result.PatchVersion}`,
+      BuildNumber: result.BuildNumber,
+    };
+  });
+}
```
Comment (mshanemc) [inline src/package1/packageDisplay.ts:30]:
```suggestion
  return results.map((result) => ({
      MetadataPackageVersionId: result.Id,
      MetadataPackageId: result.MetadataPackageId,
      Name: result.Name,
      ReleaseState: result.ReleaseState,
      Version: `${result.MajorVersion}.${result.MinorVersion}.${result.PatchVersion}`,
      BuildNumber: result.BuildNumber,
  }));
}
```

**Pull request merged**